.gitignore

```diff
@@ -3,3 +3,7 @@ fragments/runtime
 *.log
 *.retry
 __pycache__
+
+# these are temp directories for downloaded images and logs
+containers/data
```

Comment:
This isn't meant to exist normally. It could be because you started the launch script in the containers/ directory instead of the project root, though. But then I think  just "data/" should be enough as that matches in any subdir?

Reply:
I was thinking about it but I'm a bit concerned that it will be used on purpose. However, in that case comitter should spot that it's not part of the commit. So yes, I'll change that.